docparse/processing_options.mdx

```diff
@@ -4,39 +4,18 @@ description: 'Learn about the parameters you can use with Aryn DocParse'
-        * Behavior of `marked` merger:
-        For each page, it detects whether the elements are present in two columns and if so, it orders them properly with the left column appearing in the order first. Then, it drops elements that are only one token in size. It also drops elements in the top and bottom 5% of the page. Elements returned do not have a `type`. The `marked` merger always breaks across pages.
-        * Behavior of `greedy_text_element` merger:
-        Merges elements into the last most recently merged set of elements unless doing so would make its token count exceed `max_tokens`. In that case, it would keep the new element separate and start merging subsequent elements into that one, following the same rule. Merges elements on different pages, unless `merge_across_pages` is set to `False`.
-    * `max_tokens`: An integer specifying cutoff for splitting chunks that are too large. Default value is 512.
-    * `tokenizer`: A string specifying the tokenizer to use when determining how characters in a chunk are grouped. Valid values are 'openai_tokenizer', 'character_tokenizer', and 'huggingface_tokenizer'. Defaults to openai_tokenizer.
-    * `tokenizer_options`: A tree specifying the options for the chosen tokenizer. Defaults to 'model_name': 'text-embedding-3-small', which works with the OpenAI tokenizer.
-        * Available options for `openai_tokenizer`:
-            * `model_name`: Accepts all models supported by OpenAI's [tiktoken tokenizer](https://github.com/openai/tiktoken). Default is "text-embedding-3-small"
-        * Available options for `HuggingFaceTokenizer`:
-            * `model_name`: Accepts all huggingface tokenizers from the [huggingface/tokenizers repo](https://github.com/huggingface/tokenizers)
-        * `character_tokenizer` does not take any options.
-    * `merge_across_pages`: A boolean that determines whether the chunker should attempt to merge chunks on different pages. Defaults to `true`. Does not apply to the 'marked' merger, which never merges across pages.
-* `output_format`:  Defaults to `json` which yields an array called `elements` which contains the partitioned elements, represented in JSON.  If set to `markdown` the service response will instead include a field called `markdown` that contains a string representing the entire document in Markdown format.
+All of the available options are listed below, and are optional unless specified otherwise.
+<PartitionOptions/>
+<ul> <li> `pages_per_call`: This is only available when using the Partition function in Sycamore. This option divides the processing of your document into batches of pages, and you specify the size of each batch (number of pages). This is useful when running OCR on large documents.  </li> </ul>
```

Comment:
@jonfritz Do we want to talk about Sycamore within docs.aryn.ai? I thought we were only talking about Aryn's other offerings.